apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.mustache

```diff
@@ -0,0 +1,20 @@
+<script>
+    $(async () => {
+        const module = await import("pl-excalidraw-js");
+        await module.init_sketch("{{uuid}}", "{{name}}", JSON.parse(atob("{{metadata}}")));
+    });
+</script>
+
+<div>
+    <input type="hidden" name="{{name}}" id="excalidraw-input-{{uuid}}"/>
+    <style>
+        .excalidraw-root {
+            --container-width: {{width}};
+            --container-height: {{height}};
```

Comment:
Does this work correctly with multiple elements, each having different widths?

Reply:
Nice catch! I'm refactoring this as an inline style